cmd/observer/node_builder/observer_builder.go

```diff
@@ -144,9 +144,12 @@ type ObserverServiceConfig struct {
 	scriptExecutorConfig         query.QueryConfig
 	executionDataSyncEnabled     bool
 	executionDataIndexingEnabled bool
+	useLocalServiceAPI           bool
```

Comment:
Let's name this similar to other "enabled" flags
```suggestion
	localServiceAPIEnabled           bool
```